### PR TITLE
Fixing envs for credentials

### PR DESCRIPTION
**Pull request by pubudu538**
### Purpose
Fixes https://github.com/wso2/product-microgateway/issues/2446

### Issues

Fixes #

### Automation tests
 - Unit tests added: Yes/No
 - Integration tests added: Yes/No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: adapter/internal/oasparser/model/api_env.go [hunk 1]
```diff
@@ -79,8 +79,10 @@ func retrieveEndpointsFromEnv(apiHashValue string) ([]Endpoint, []Endpoint) {
 //RetrieveEndpointBasicAuthCredentialsFromEnv retrieve endpoint security credentials from env variables
 func RetrieveEndpointBasicAuthCredentialsFromEnv(apiHashValue string, keyType string, endpointSecurity EndpointSecurity) EndpointSecurity {
 	// production Endpoint security
-	endpointSecurity.Username = resolveEnvValueForEndpointConfig(apiHashValue+"_"+keyType+"_basic_username", endpointSecurity.Username)
-	endpointSecurity.Password = resolveEnvValueForEndpointConfig(apiHashValue+"_"+keyType+"_basic_password", endpointSecurity.Password)
+	endpointSecurity.Username = resolveEnvValueForEndpointConfig("api_"+apiHashValue+"_"+keyType+
+		"_basic_username", endpointSecurity.Username)
+	endpointSecurity.Password = resolveEnvValueForEndpointConfig("api_"+apiHashValue+"_"+keyType+
+		"_basic_password", endpointSecurity.Password)
```
Comment (praminda) [inline adapter/internal/oasparser/model/api_env.go:85]:
```suggestion
	endpointSecurity.Username = resolveEnvValueForEndpointConfig("api_" + apiHashValue + "_" + keyType + 
		"_basic_username", endpointSecurity.Username)
	endpointSecurity.Password = resolveEnvValueForEndpointConfig("api_" + apiHashValue + "_" + keyType + 
		"_basic_password", endpointSecurity.Password)
```

**Review by AmaliMatharaarachchi — APPROVED** [review]

File: adapter/internal/oasparser/model/api_env.go (hunk 1 above)
Comment (pubudu538) [reply adapter/internal/oasparser/model/api_env.go]: Fixed

Comment (praminda) [reply adapter/internal/oasparser/model/api_env.go]: im sorry, it seems go doesn't like to have these spaces. vscode automatically reverts the change. shall we undo the commit

Comment (pubudu538) [reply adapter/internal/oasparser/model/api_env.go:85]: fixed.

**Review by praminda — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[failed] dev-deployment : 20211215.32

**Comment by choreo-cicd** [comment]
[succeeded] dev-deployment : 20211215.43

**Pull request merged**